### ISSUE TITLE
NVDA announces the SVG container label multiple times

### ISSUE THREAD

**Issue by LucyLiYY** [highcharts/highcharts#18087]
#### Expected behaviour
NVDA should announce the svg container label (svgContainerLabel) only once when entering the chart

#### Actual behaviour
NVDA announces the SVG container label multiple times when moving to each data point.

#### Live demo with steps to reproduce

https://codepen.io/LucyLi123/pen/ExRGLqy

#### Product version

Latest highchart

**Comment by oysteinmoseng** [comment]
I'm afraid there is not much we can do about this on our end, as it is down to the quirks of NVDA and SVG accessibility.

More navigation modes based on ARIA applications are planned in the future, where we can avoid this.

As a workaround for now, setting `svgContainerLabel` to an empty string will remove it from the chart.

I am closing this for now, but feel free to comment to reopen if you have further information to add.
